docs/resources/cce_node_pool.md

```diff
@@ -102,6 +102,8 @@ The following arguments are supported:
 
 * `labels` - (Optional, Map, ForceNew) Tags of a Kubernetes node, key/value pair format. Changing this parameter will create a new resource.
 
+* `tags` - (Optional, Map, ForceNew) Tags of a VM node, key/value pair format. Changing this parameter will create a new resource.
+
```

Comment:
According to the Go programming specification, a single line of code should not exceed 120 characters.
The (All) document content should be adjusted later.

Reply:
fixed